gammapy/modeling/models/core.py

```diff
@@ -428,6 +428,21 @@ def to_dict(self, full_output=False):
         else:
             return {"components": models_data}
 
+    def to_parameters_table(self):
+        """Convert Models parameters to an astropy Table."""
+        table = self.parameters.to_table()
+        #Warning: splitting of parameters will break is source name has a "." in its name.
+        model_name = [name.split(".")[0] for name in self.parameters_unique_names]
+        table.add_column(model_name, name='model', index=0)
+        self._table_cached = table
+        return  table
+
+    def set_parameters_from_table(self, t):
```

Comment:
One last minor thing: maybe rename here to `.update_parameters_from_table()`, the we have a uniform `.update_from_something()` pattern...

Reply:
done